### PR TITLE
Sf3/PHP7 update

### PR DESCRIPTION
**Pull request by kaltun**

### CODE REVIEW

File: Form/TranslatableType.php [hunk 1]
```diff
@@ -12,7 +12,9 @@
 namespace Nfq\AdminBundle\Form;
 
 use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\HiddenType;
 use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
 use Symfony\Component\OptionsResolver\OptionsResolverInterface;
```
Comment (N3m1s) [inline Form/TranslatableType.php]: `use Symfony\Component\OptionsResolver\OptionsResolverInterface` is not used anymore?

File: Resources/config/services.yml [hunk 2]
```diff
@@ -55,7 +55,7 @@ services:
     nfq_admin.form_ext.tiny_mce:
           class: Nfq\AdminBundle\Form\Extension\TinyMCEExtension
           tags:
-              - { name: form.type_extension, alias: textarea }
+              - { name: form.type_extension, extended_type: Symfony\Component\Form\Extension\Core\Type\TextareaType }
```
Comment (N3m1s) [inline Resources/config/services.yml]: to awoid BC, both, `alias` end `extended_type` should be here

File: composer.json [hunk 3]
```diff
@@ -5,7 +5,7 @@
     "type": "symfony-bundle",
     "require": {
         "php": ">=5.4",
-        "symfony/symfony": "~2.3",
```
Comment (N3m1s) [inline composer.json:8]: Looking at the changes I seems that they would work with Symfony 2.7, 2.8 too, so the version should be 
`"symfony/symfony": "~2.7|~3.0"`

File: composer.json [hunk 4]
```diff
@@ -5,7 +5,7 @@
     "type": "symfony-bundle",
     "require": {
         "php": ">=5.4",
```
Comment (N3m1s) [inline composer.json]: The use of `::class` implies that the minimum version for PHP should be 5.5

**Review by N3m1s — CHANGES_REQUESTED** [review]

File: Form/TranslatableType.php (hunk 1 above)
Comment (kaltun) [reply Form/TranslatableType.php]: No, OptionsResolverInterface is deprecated from symfony 2.7.

File: composer.json (hunk 3 above)
Comment (kaltun) [reply composer.json:8]: changed

File: composer.json (hunk 4 above)
Comment (kaltun) [reply composer.json]: thank you for the notice!!!

File: Resources/config/services.yml (hunk 2 above)
Comment (kaltun) [reply Resources/config/services.yml]: fixed

File: Form/TranslatableType.php (hunk 1 above)
Comment (N3m1s) [reply Form/TranslatableType.php]: So why it is there, in use's??

**Pull request merged**